optaplanner-quarkus-integration/optaplanner-quarkus/deployment/src/main/resources/dev-templates/solverConfig.html

```diff
@@ -0,0 +1,47 @@
+<!--
+~ Copyright 2021 Red Hat, Inc. and/or its affiliates.
+~
+~ Licensed under the Apache License, Version 2.0 (the "License");
+~ you may not use this file except in compliance with the License.
+~ You may obtain a copy of the License at
+~
+~      http://www.apache.org/licenses/LICENSE-2.0
+~
+~ Unless required by applicable law or agreed to in writing, software
+~ distributed under the License is distributed on an "AS IS" BASIS,
+~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+~ See the License for the specific language governing permissions and
+~ limitations under the License.
+-->
+{#include main}
+
+{#style}
+.solver-config {
+white-space: pre-wrap;
+}
+{/style}
+
+{#title}Solver Configuration{/title}
+
+<!--
+  ~ Copyright 2021 Red Hat, Inc. and/or its affiliates.
+  ~
+  ~ Licensed under the Apache License, Version 2.0 (the "License");
+  ~ you may not use this file except in compliance with the License.
+  ~ You may obtain a copy of the License at
+  ~
+  ~      http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  -->
+
+{#body}
+<div class="solver-config">
+    {info:solverConfigProperties.effectiveSolverConfig}
```

Comment:
I think this the XML should be preformatted. (HTML `<pre>`)
Better yet, highlighted if there is a way to do that in the Dev UI.

Reply:
Changed the `div` to `pre`, I don't think we can get "automatic highlighting" via Qute/Dev UI (beside adding our own script).